schema.h

```diff
@@ -47,53 +62,77 @@ struct User : yoda::Padawan {
   template <typename A>
   void serialize(A& ar) {
     Padawan::serialize(ar);
-    ar(CEREAL_NVP(score));
+    ar(CEREAL_NVP(level), CEREAL_NVP(score));
   }
 };
 
-struct UIDTokenPair : yoda::Padawan {
-  UID uid = UID::INVALID;
+// AuthKey structure defines generic authentication key, which consists of a pair of `std::string`s.
+// `key1` and `key2` may be used in different ways, depending on the authentication type.
+// For `type = AUTH_TYPE::IOS`:
+//  - key1 = Device ID
+//  - key2 = Application key, a random number, generated once in a lifetime in iOS application on its first
+//  launch.
+struct AuthKey {
+  std::string key1 = "";
+  std::string key2 = "";
+  AUTH_TYPE type = AUTH_TYPE::UNDEFINED;
```

Comment:
I'd move `type` to the top, and rename `key1`, `key2`, into `did`, and `did` (`device_key`, and `random_key`).